internal/ui/ui_glfw.go

```diff
@@ -677,7 +828,17 @@ func (u *userInterfaceImpl) createWindow(width, height int) error {
 	if err != nil {
 		return err
 	}
+
+	// Set our target monitor if provided. This is required to prevent an initial window flash on the default monitor.
+	if monitor >= 0 && len(monitors) > monitor {
```

Comment:
If `monitor` should never be a negative value, or `len(monitors)` can be <= 0, this should cause panic.

Reply:
Monitor can be a negative value, as that is used to represent that we don't care which monitor to use (it uses default ebiten behavior for choosing a monitor). I think the `len(monitors)` check can be safely removed, as it shouldn't be possible to pass a monitor index that exceeds the monitors array.

For more info: we use the `userInterfaceImpl.initWindowMonitor` field to populate the desired initial monitor. This defaults to `glfw.DontCare` which is `-1`.

I have adjusted it to compare against `glfw.DontCare`, as that should be fine.

Reply:
Is `glfw.DontCare` appropriate, or should a new const be made, such as `DefaultMonitor`?

Reply:
Let's use `glfw.DontCare`.